src/schema/users.ts

```diff
@@ -2190,6 +2175,20 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         coresRole: userCoresRole,
       };
     },
+    notificationSettings: async (
+      _,
+      __,
+      ctx: AuthContext,
+    ): Promise<UserNotificationFlags> => {
+      const user = await ctx.con.getRepository(User).findOne({
+        where: { id: ctx.userId },
+        select: ['notificationFlags'],
+      });
+      return {
+        ...DEFAULT_NOTIFICATION_SETTINGS,
+        ...user?.notificationFlags,
+      };
```

Comment:
since notification flags are nested, it would not be deep merged so I don't see any point in having default settings, but non blocking, something to maybe adjust down the line